LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/LoggerHelper.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace Logger
+{
+    using System;
+    using System.Collections.Generic;
+    using LoRaWan;
+    using Microsoft.Extensions.Logging;
+
+    internal static class LoggerHelper
+    {
+        /// <summary>
+        /// Prefixes the message with predefined scope values.
+        /// Right now we only support DevEUI and DevAddr, but could be extended to
+        /// include others (StationEUI, Gateway, etc).
+        /// If the message is already prefixed with the DevEUI/DevAddr, we don't
+        /// add it again.
+        /// </summary>
+        /// <param name="message">The already formatted message</param>
+        /// <returns></returns>
```

Comment:
Can we update the docs? Specifically I think we support stationEUI as a scope right?

Reply:
Can we add a note about the order of scopes defined being important?

Reply:
Good catch, updated with 8a39f0caaa8319ef0fefa1f28a69221c7bb5b174